polkadot/xcm/src/lib.rs

```diff
@@ -443,6 +443,16 @@ impl<C> IntoVersion for VersionedXcm<C> {
 	}
 }
 
+impl<C> IdentifyVersion for VersionedXcm<C> {
```

Comment:
Hmm, this is already implemented for types created with `versioned_type`. For some reason `VersionedXcm` is not. Making a note here to investigate why in a later issue/pr.